app/scripts/proactive/templates/job-template.html

```diff
@@ -21,7 +21,7 @@
     <% if (job["Generic Info"] && job["Generic Info"].length>0) { %>
     <genericInformation>
     <% _.each(job["Generic Info"], function(gi) { %>
-            <info name="<%=gi["Property Name"]%>" value="<%=gi["Property Value"]%>"/>
+            <info name="<%=gi["Property Name"]%>" value="<%= _.escape(gi["Property Value"]) %> "/>

```

Comment:
We put a json object inside the GI. the EXECUTION_CALENDARS object. That introduces thinks like < {} which will be interpreted as XML. That breaks the parsing of the workflow inside the Execution Scheduler